compiler/src/iree/compiler/Codegen/Common/StripCompilationInfoPass.cpp

```diff
@@ -69,19 +69,17 @@ struct StripAttentionOpCompilationInfo final
       eraseLoweringConfig(attentionOp);
     }
 
-    DictionaryAttr decompositionConfig =
-        attentionOp.getDecompositionConfigAttr();
-    if (decompositionConfig) {
-      decompositionConfig = DictionaryAttr::get(
+    if (DictionaryAttr decompositionConfig =
+            attentionOp.getDecompositionConfigAttr()) {
+      DictionaryAttr modifiedDecompositionConfig = DictionaryAttr::get(
```

Comment:
nit: this is a bit of a mouthful and I don't think we need to be that descriptive here
```suggestion
      DictionaryAttr newConfig = DictionaryAttr::get(
```